pkg/osquery/runtime/osquery_instance_history/update.go

```diff
@@ -0,0 +1,35 @@
+package osquery_instance_history
+
+import "sync"
+
+var mutex sync.Mutex
+var currentHistory = &History{}
```

Comment:
Does it make sense to put the mutex into History and move the locks there? Might not matter

Reply:
it's cleaner to have the History handle the locks, I'll do that

Reply:
well I put the locks on the history struct, but still letting package exported funcs handling locking, got to confusing to manage the locks in the unexported functions